src/control/AxesController.ts

```diff
@@ -378,15 +379,16 @@ class AxesController {
 
     if (duration === 0) {
       const flicking = getFlickingAttached(this._flicking);
-      const camera = flicking.camera;
 
       animate();
 
-      const newPos = flicking.circularEnabled
-        ? circulatePosition(position, camera.range.min, camera.range.max)
-        : position;
-
-      axes.axisManager.set({ [AXES.POSITION_KEY]: newPos });
+      if (flicking.moveType !== MOVE_TYPE.STRICT) {
+        const camera = flicking.camera;
+        const newPos = flicking.circularEnabled
+          ? circulatePosition(position, camera.range.min, camera.range.max)
+          : position;
+        axes.axisManager.set({ [AXES.POSITION_KEY]: newPos });
+      }
```

Comment:
It's a good practice to not include some child-class-specific code inside the general loop.

So, my suggestion is:
As this code is quite specific for `moveType: "strict"`, what about updating `position` after this in `StrictControl` instead?
One good place to put it is `_animateToPosition` I guess, you can try to inherit it & check whether the `position` is inside the correct range and update it.